src/app/garfile/manifest/editperson/index.njk

```diff
@@ -30,211 +30,22 @@
-          <label class="govuk-label govuk-label--m" for="first_name">Given Name(s)</label>
-           {{ m.error_message('first_name', errors) }}
-          <input class="govuk-input" id="first_name" name="first_name"
-          autocomplete="false" type="text" value="{{personDetails.firstName}}" />
-        </div>
-
-        <div class="govuk-form-group {{ m.form_group_class('dob', errors) }}">
-          <fieldset class="govuk-fieldset" aria-describedby="dateOfBirth-hint" role="group">
-          <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
-            Date of Birth
-          </legend>
-          <span class="govuk-hint" id="dateOfBirth-hint">For example, 31 3 1980</span>
-          <div class="form-date" id="dob">
-          {{ m.error_message('dob', errors) }}
-
-            <div class="govuk-date-input" id="dob">
-              <div class="govuk-date-input__item">
-                <div class="govuk-form-group">
-                  <label class="govuk-label govuk-date-input__label" for="dobday">Day</label>
-                  {{ m.error_message('dobday', errors) }}
-                  <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="dobday" name="dobday" type="number" value="{{cookie.dateSlice('day', personDetails.dateOfBirth)}}" pattern="[0-9]*">
-                </div>
-              </div>
-
-              <div class="govuk-date-input__item">
-                <div class="govuk-form-group">
-                  <label class="govuk-label govuk-date-input__label" for="dobmonth">Month</label>
-                  {{ m.error_message('dobmonth', errors) }}
-                <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="dobmonth" name="dobmonth" type="number" value="{{cookie.dateSlice('month', personDetails.dateOfBirth)}}" pattern="[0-9]*">
-                </div>
-              </div>
-
-              <div class="govuk-date-input__item">
-                <div class="govuk-form-group">
-                  <label class="govuk-label govuk-date-input__label" for="dobyear">Year</label>
-                  {{ m.error_message('dobyear', errors) }}
-                  <input class="govuk-input govuk-date-input__input govuk-input--width-4" id="dobyear" name="dobyear" type="number" value="{{cookie.dateSlice('year', personDetails.dateOfBirth)}}" pattern="[0-9]*">
-                </div>
-              </div>
-        </fieldset>
-      </div>
-
-    <div class="govuk-form-group {{ m.form_group_class('persontype', errors) }}">
-      <label class="govuk-label govuk-label--m" for="persontype">Person Type</label>
-      <span class="govuk-hint">
-        For example, Captain, Crew or Passenger. First
-        Office / Co-Pilot should be identified as Crew.
-      </span>
-      {{ m.error_message('persontype', errors) }}
-      <select class="govuk-select" id="persontype" name="persontype">
-        {% for person in travepersontype %}
```

Comment:
Travel Document Details field is not visibile in the "Add person details" page anymore.

Reply:
The page has been changed to be the same as the saved person page, this was always the intention. The current manifest page's layout does not facilitate the changing of content as suggested